### ISSUE TITLE
Manba-YOLO

### ISSUE THREAD

**Issue by xsa12345** [HZAI-ZJNU/Mamba-YOLO#36]
Is Mamba-yolo based on YOLOv8? Is it still based on the YOLO architecture?

**Comment by EthanW-coder** [comment]
Yes, Mamba-YOLO is based on the official YOLOv8 ultralytics implementation, and it's worth mentioning that we're working on a version based on mmdet.

> Mamba-yolo 基于 YOLOv8 吗？它仍然基于 YOLO 架构吗？
